docs/content/quickstart/quick-start-docker.md

```diff
@@ -48,17 +51,17 @@ SENDGRID_API_KEY="some_sendgrid_key"
 ..
 ```
 
-5. Start your server
+### 5. Start your server
 
 ```bash
 docker-compose up --build
 ```
 
-We will use docker-compose and Docker to start up our development environment. Running the above command will do the following:
+Running the above command does the following:
```

Comment:
Add the following note before this:

~~~
:::note

If you get the error `ERROR: for postgres  Cannot start service postgres: Ports are not available`, change the ports used for PostgreSQL in `docker-compose.yml` to something like this:

```yml
  postgres:
    ports:
      - "5433:5433"
```

:::
~~~